src/torcontrol.cpp

```diff
@@ -608,7 +608,9 @@ void TorController::disconnected_cb(TorControlConnection& conn)
     service = CService();
     if (!reconnect)
         return;
-    LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+
+    LogPrint("tor", "tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
```

Comment:
Yes, it's one or the other:
- Make the logic determine whether there was a connection, report if there was one and it was lost, report a different message if we're re-trying to connect because connection failed
- Change to a blanket message that covers both cases

If not, people that have `-debug=tor` enabled can still complain that the message is not correct.
The second is just a matter of changing a message so should be done in this PR, IMO.